app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizardSelectors.js

```diff
@@ -14,3 +16,9 @@ export const planWizardFormFilter = form => ({
   planWizardAdvancedOptionsStep: form.planWizardAdvancedOptionsStep,
   planWizardScheduleStep: form.planWizardScheduleStep
 });
+
+export const getCurrentTargetProvider = (form, transformationMappings) => {
+  const mappingId = form.planWizardGeneralStep.values.infrastructure_mapping;
```

Comment:
I'm sorta torn about how specific `form` has to be in order for this helper to work.  Seems like it would be better for it to take an `id` as the argument...  but for now, I can't immediately think of any use cases for this outside of PlanWizard, so, not a blocker.

Reply:
Oh wait, this is a redux selector.  Nevermind, my bad.  As you were :)